runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java

```diff
@@ -630,16 +630,26 @@ public void makeOverlayHint(PanelComponent panelComponent, ClueScrollPlugin plug
 				inventory = new Item[0];
 			}
 
+			Item[] combined = new Item[equipment.length + inventory.length];
+			System.arraycopy(equipment, 0, combined, 0, equipment.length);
+			System.arraycopy(inventory, 0, combined, equipment.length, inventory.length);
+
 			for (ItemRequirement requirement : getItemRequirements())
 			{
 				boolean equipmentFulfilled = requirement.fulfilledBy(equipment);
 				boolean inventoryFulfilled = requirement.fulfilledBy(inventory);
+				boolean combinedFulfilled = false;
```

Comment:
Assign this in all cases and remove the `inventoryFulfilled` check, as it is not needed.